governance/handler.go

```diff
@@ -168,12 +168,28 @@ func (gov *Governance) checkKey(k string) bool {
 	return false
 }
 
+func (gov *Governance) checkValue(key string, val interface{}) bool {
+
+	if key == "istanbul.committeesize" && val == uint64(0) {
+		return false
+	}
+	if key == "reward.minimumstake" {
+		if v, ok := new(big.Int).SetString(val.(string), 10); ok {
+			if v.Cmp(common.Big0) <= 0 {
+				return false
+			}
+		}
+	}
```

Comment:
Why do you check only for these 2 governance items?
Are other items good to go with no value-check in this function?

Reply:
Other items can be discussed and added if there are problems.